plugins/woocommerce/client/legacy/js/admin/meta-boxes-product.js

```diff
@@ -734,7 +707,9 @@ jQuery( function ( $ ) {
 	$( '.product_attributes' ).on(
 		'click',
 		'button.add_new_attribute',
-		function () {
+		function ( event ) {
+			event.preventDefault();
```

Comment:
What are we preventing in this case?

Reply:
Before, the function was returning false, which prevented form submission and event propagation. I can add a comment to mention that it is preventing form submission.

Reply:
I'm ok with adding the comment!

Reply:
Added a comment.